src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
+| **Tool** | [Rust SDK](https://docs.rs/soroban-sdk/latest/soroban_sdk/) | Supports writing programs for Stellar smart contracts |
+
+### Stellar for smart contract developers
+
+| **Contract Devs** |  |  |
+| :-: | --- | --- |
+| **Learn** | [Stellar smart contracts overview](https://developers.stellar.org/docs/build/smart-contracts/overview) | Learn the essentials about writing smart contracts on Stellar |
+| **Learn** | [Migrating from EVM networks](https://developers.stellar.org/docs/learn/migrate) | Stellar is not an EVM network, learn about how Stellar differs from EVM networks here |
+| **Learn** | [Storage types](https://developers.stellar.org/docs/learn/encyclopedia/storage/state-archival#contract-data-type-descriptions) | There are three different storage types for smart contract data, learn about them here |
+| **Guide** | [Getting started guide](https://developers.stellar.org/docs/build/smart-contracts/getting-started) | The go-to guide for getting setup to write smart contracts on Stellar |
+| **Guide** | [Example contracts](https://developers.stellar.org/docs/build/smart-contracts/example-contracts) | These contracts are good starting points for understanding best practices and writing your own contracts on Stellar |
+| **Guide** | [General contract guides](https://developers.stellar.org/docs/build/guides/conventions) | Various guides that highlight the norms seen in contract development |
+| **Tool** | [Stellar CLI](https://developers.stellar.org/docs/build/guides/cli) | The command line interface to Stellar smart contracts |
+| **Tool** | [Rust SDK](https://docs.rs/soroban-sdk/latest/soroban_sdk/) | Supports writing programs for Stellar smart contracts |
+
+| **Contract Devs (optional pages)** |  |  |
+| :-: | --- | --- |
+| **Learn** | [Trustlines](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/accounts#trustlines) | Trustlines are unique to Stellar and are an explicit opt-in for an account to hold and trade a particular asset. |
+| **Guide** | [Upgrading contracts](https://developers.stellar.org/docs/build/guides/conventions/upgrading-contracts) | You don’t need a proxy pattern to upgrade a contract (like in EVM), you can upgrade the Wasm bytecode directly without changing the address |
+
+### Stellar for ramps (anchors)
+
+| **Ramps** |  |  |
+| :-: | --- | --- |
+| **Learn** | [Anchor overview](https://developers.stellar.org/docs/learn/fundamentals/anchors) | Stellar ramps are called "anchors", learn about them in this overview section |
+| **Learn** | [SEP-6 and SEP-24](http://developers.stellar.org/docs/learn/fundamentals/anchors#using-sep-6-programmatic-deposit-and-withdrawal-versus-sep-24-hosted-deposit-and-withdrawal) | You can set up an anchor using SEP-6 or SEP-24, learn about the differences here |
+| **Tutorial** | [Anchor Platform](https://developers.stellar.org/platforms/anchor-platform) | Set up an anchor using the SDF-maintained Anchor Platform |
```

Comment:
What's the difference between `Tutorials` and `Guides`?

Reply:
A tutorial teaches the user a new concept (or series of concepts) with steps on how to accomplish a goal. Tutorials take you through a process step-by-step with learning along the way. A guide is usually focused on achieving a particular task and can assume some level of prerequisite knowledge about a subject.

You can see the differences in something like the entire Anchor Platform section, which walks the reader through what the Anchor Platform is, how to set it up, the different SEPs, etc. It's very comprehensive. Whereas a guide is for accomplishing a very specific task: like, "Extend a deployed contract's TTL".